src/ci/run.sh

```diff
@@ -74,6 +74,12 @@ retry make prepare
 travis_fold end make-prepare
 travis_time_finish
 
+travis_fold start bootstrap-test
+travis_time_start
+python $SRC/src/bootstrap/bootstrap_test.py
```

Comment:
Let's add this as a target to the makefile. I think you can edit `Makefile.in` to do so, and then we can use the `CFG_PYTHON` and `CFG_SRC_DIR` variables. `check-bootstrap` seems fine.